CHANGELOG.md

```diff
@@ -1,5 +1,337 @@
 # Changelog
 
+## 🔎 1.1.0
+
+Wrangler 1.1.0 includes a number of improvements to documentation and project stability, including:
+
+- ### Security
+
+  - **Change global config perm - [xtuc], [pull/286]**
+
+  This PR improves the security of Wrangler's global config file, by only allowing the user using Wrangler to view the file. **For existing Wrangler users, please run `wrangler config` again on your machine!** This will fix the permissions of your global config file to be scoped to your user account.
+
+  [xtuc]: https://github.com/xtuc
+  [pull/286]: https://github.com/cloudflare/wrangler/pull/286#event-2516759398
+
+- ### Features
+
+  - **feat: Support KV Namespace Configuration - [ashleymichal], [pull/334]**
+
+  Wrangler now supports using KV namespaces in your project! We now support defining [Workers KV][kv] namespaces in your `wrangler.toml` configuration file. The `kv-namespaces` key requires setting a `binding` (the representation of your namespace in your code) and `id`, the namespace ID:
```

Comment:
i would add a sentence that says something like- to use KV, create a namespace in the UI. then add to wrnagler.toml. this is a great place to hint at what's to come and link to the KV milestone